docs/src/Box.doc.js

```diff
@@ -1,24 +1,157 @@
 // @flow strict
 import React, { type Node } from 'react';
-import { Box, Button } from 'gestalt';
+import { Box } from 'gestalt';
 import PropTable from './components/PropTable.js';
 import Example from './components/Example.js';
-import Combination from './components/Combination.js';
+import CombinationNew from './components/CombinationNew.js';
+
 import PageHeader from './components/PageHeader.js';
 import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Box"
-    description={`In the darkest night, Box will rise to bring the light. The Lloyd has spoken.
-
-&mdash; Anon _(Winning Box Haiku, 2017)_`}
+    description="Box is a low level component that can be used to build the foundation of pretty much anything. Using Box allows you to focus on the important content, without worrying about the pixel details."
```

Comment:
I agree the previous description was... a joke! I still feel we don't describe Box to its essence which is a div. 

I have mix feelings about: `Using Box allows you to focus on the important content, without worrying about the pixel details.` It's describing the intention, but not describing it.